--- a/src/main/java/views/graphical/screens/GameScreen.java
+++ b/src/main/java/views/graphical/screens/GameScreen.java
@@ -504,6 +504,11 @@
     private void saveAndExit() {
         removeModal();
 
+        if (gameHud != null) {
+            gameHud.hideGameHud();
+            gameHud.remove();
+        }
+
         Gdx.app.postRunnable(
                 () -> game.showScreen(
                         new MainMenuScreen(game)
--- a/src/main/java/views/graphical/screens/BaseScreen.java
+++ b/src/main/java/views/graphical/screens/BaseScreen.java
@@ -2,6 +2,7 @@
 
 import com.badlogic.gdx.InputProcessor;
 import com.badlogic.gdx.ScreenAdapter;
+import com.badlogic.gdx.utils.ScreenUtils;
 import com.badlogic.gdx.scenes.scene2d.Stage;
 import com.badlogic.gdx.scenes.scene2d.ui.Skin;
 import com.badlogic.gdx.utils.viewport.ExtendViewport;
@@ -43,6 +44,8 @@
     public void render(float delta) {
         float safeDelta = Math.min(delta, 0.1f);
 
+        ScreenUtils.clear(0f, 0f, 0f, 1f);
+
         stage.act(safeDelta);
         stage.draw();
     }
