--- a/src/main/java/views/graphical/ui/StartGameMenuPopup.java
+++ b/src/main/java/views/graphical/ui/StartGameMenuPopup.java
@@ -37,7 +37,7 @@
 
         TextureRegion innerBackgroundRegion =
                 game.getTextureBank().region(
-                        "IMAGE_UI_CARDS_CARD_TABLE_FRAME"
+                        "IMAGE_UI_VASEBREAKER_ENDLESS_BG"
                 );
 
         if (circle == null) {
@@ -68,10 +68,10 @@
                 )
         );
         innerCard.pad(
-                12f,
-                24f,
-                24f,
-                24f
+                18f,
+                28f,
+                18f,
+                28f
         );
 
         Table titleTable = new Table();
@@ -81,10 +81,10 @@
                 )
         );
         titleTable.pad(
-                7f,
-                28f,
-                7f,
-                28f
+                10f,
+                26f,
+                10f,
+                26f
         );
 
         Label titleLabel =
@@ -103,9 +103,10 @@
                 .fillX();
 
         innerCard.add(titleTable)
-                .expandX()
-                .fillX()
-                .padBottom(10f)
+                .width(410f)
+                .center()
+                .padTop(4f)
+                .padBottom(12f)
                 .row();
 
         Label levelLabel =
@@ -125,7 +126,7 @@
         innerCard.add(levelLabel)
                 .expandX()
                 .fillX()
-                .padBottom(12f)
+                .padBottom(16f)
                 .row();
 
         Table objectivesTable = new Table();
@@ -158,13 +159,10 @@
         }
 
         innerCard.add(objectivesTable)
+                .width(430f)
                 .left()
-                .expandX()
-                .fillX()
-                .padTop(4f)
-                .padLeft(14f)
-                .padRight(14f)
-                .padBottom(10f)
+                .padTop(2f)
+                .padBottom(12f)
                 .row();
 
         BitmapFont newFont =
@@ -205,19 +203,19 @@
 
         this.getContent()
                 .add(innerCard)
-                .width(520f)
+                .width(500f)
                 .pad(
+                        16f,
                         18f,
-                        18f,
-                        6f,
+                        4f,
                         18f
                 )
                 .row();
 
         this.getContent()
                 .add(continueButton)
-                .padTop(-2f)
-                .padBottom(-34f)
+                .padTop(4f)
+                .padBottom(-26f)
                 .align(Align.center);
 
         this.pack();
@@ -248,20 +246,21 @@
         objectiveLabel.setColor(
                 Color.BROWN
         );
-
+        objectiveLabel.setAlignment(Align.left);
         objectiveLabel.setWrap(true);
 
         objectivesTable.add(bulletImage)
-                .size(16f, 16f)
+                .size(14f, 14f)
                 .top()
-                .padTop(3f)
-                .padRight(10f);
+                .padTop(4f)
+                .padRight(12f);
 
         objectivesTable.add(objectiveLabel)
                 .left()
                 .expandX()
                 .fillX()
-                .padBottom(7f)
+                .width(400f)
+                .padBottom(12f)
                 .row();
     }
 }
